.circleci/workflows.yml

```diff
@@ -173,11 +173,11 @@ jobs:
             - /home/circleci/.brownie
             - /home/circleci/.solcx
 
-  test-agent:
+  test-beamer:
     executor: vm
     steps:
       - initialize-environment
-      - run: poetry run brownie test beamer/tests/agent -v -G --cov beamer --cov-report=term
+      - run: poetry run brownie test beamer/tests/{agent,health} -v -G --cov beamer --cov-report=term
```

Comment:
Why does it have to be agent & health? Can't we just run with `beamer/tests`?

Reply:
The health tests just run in few seconds - I don't think that a new workflow will speed up things. Let's leave it like this.